src/test/java/de/tum/in/www1/artemis/ExamIntegrationTest.java

```diff
@@ -70,22 +75,108 @@
     private Exam exam1;
 
     @BeforeEach
-    public void initTestCase() {
+    public void initTestCase() throws URISyntaxException {
         users = database.addUsers(4, 5, 1);
         course1 = database.addEmptyCourse();
         course2 = database.addEmptyCourse();
         exam1 = database.addExam(course1);
+        jiraRequestMockProvider.enableMockingOfRequests();
+        jiraRequestMockProvider.mockAddUserToGroup(Set.of(course1.getStudentGroupName()));
     }
 
     @AfterEach
     public void resetDatabase() {
         database.resetDatabase();
     }
 
+    @Test
+    @WithMockUser(username = "instructor1", roles = "INSTRUCTOR")
+    public void registerUsersInExam() throws Exception {
+
+        var exam = createExam();
+        var savedExam = examRepository.save(exam);
+        var student1 = database.getUserByLogin("student1");
+        var student2 = database.getUserByLogin("student2");
+        var student3 = database.getUserByLogin("student3");
+        var registrationNumber1 = "1234567";
+        var registrationNumber2 = "2345678";
+        var registrationNumber3 = "3456789";
+        var registrationNumber3WithType = "3456789" + "0";
```

Comment:
This should be `Typo` instead of `Type`, also in the 2 other occurrences. Also, we could probably replace `"3456789"` with `registrationNumber3`.

Reply:
done, thanks!